### ISSUE TITLE
conda PDL install fails

### ISSUE THREAD

**Issue by jknoel** [smog-server/OpenSMOG#17]
The instructions you give all work except for 

(smog2.4) DAUMKE59:git jnoel$ conda install -c eumetsat perl-pdl
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - perl-pdl

Current channels:

  - https://conda.anaconda.org/eumetsat/osx-64
  - https://conda.anaconda.org/eumetsat/noarch
  - https://conda.anaconda.org/conda-forge/osx-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.anaconda.com/pkgs/main/osx-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/osx-64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.

**Comment by contessoto** [comment]
Hello,
It seems to be a conflict in the environment.
I was able to reproduce the conflict when I did not create a new environment.
Please, try to follow these steps again but first, update your conda installation (conda update --prefix /path/to/anaconda3/ anaconda): https://opensmog.readthedocs.io/en/latest/GettingStarted/install.html#installing-smog2

**Comment by contessoto** [comment]
Another possible reason for the issue might be the OS:
It seems that the Perl-Pdl package is designed for Linux only when installing using conda:
https://anaconda.org/Eumetsat/perl-pdl

**Comment by jknoel** [comment]
Probably you are right that it is because the perl-pdl you suggest is for linux. I have tried various things and can't get PDL to install within the conda environment. The most obvious solution would be 

perl -MCPAN -e 'install PDL'

But there are errors. First of which is that it doesn't find /usr/bin/clang, but I surpassed that and then it becomes more mysterious

Has anyone successfully built it on mac from scratch? If not, maybe say that these instructions are for linux and that on MacOS the conda perl route isn't the best?
